core/README.md

```diff
@@ -81,16 +66,31 @@ that declare different conformance classes. This is useful when an entire catalo
 support the *STAC API - Item Search* conformance class, perhaps because it uses multiple databases to store items,
 but sub-catalogs whose items are all in one database can support search.
 
-A STAC API conforming to the *STAC API - Browseable* conformance class
-(<https://api.stacspec.org/v1.0.0-rc.2/browseable>) must be structured such that
-all Items in the catalog can be accessed by following `child` and `item` link relations. This is a more significant
-constraint than a STAC API without this conformance class or a STAC Catalog that is available over HTTP but does not
-implement STAC API, neither of which have any guarantee regarding the reachability of Items. This conformance 
-class is used to signal to users that they can fully navigate to all available Items using a UI (like [STAC Browser](https://github.com/radiantearth/stac-browser), 
-and also makes it clear to crawlers that they can reach everything by following catalog links. 
+## Browseable
+
+The Browseable conformance class (<https://api.stacspec.org/v1.0.0-rc.2/browseable>) provides a formal way
+to advertise that all Items in the Catalog presented by a STAC API may be reached by following `child` and
+`item` link relations. In a non-API STAC Catalog, all items must be reachable via these relations to be
+considered in the catalog. With a STAC API Catalog, items are considered to be in the catalog if they are
+accessible via search operations. However, a STAC API Catalog may also have child and item link relations.
```

Comment:
I have a couple of questions:
- While I like this way of thinking, but what about APIs that don't support item search?
- Would it in this case mean that all items can be reached via item/child links that are part of STAC API - Features?
- Should Browsable be used if an API only implements STAC API - Collections and Collection Search and exposes all Collections via child links (but has no items at all)?

Reply:
>     * While I like this way of thinking, but what about APIs that don't support item search?

Good point. I'll clarify this language, since it should be considered in the catalog of it's accessible via link relations **or** search via /search or /items
 
>     * Would it in this case mean that all items can be reached via item/child links that are part of STAC API - Features?

Yes -- I clarified that search includes both /search and /items

>     * Should Browsable be used if an API only implements STAC API - Collections and Collection Search and exposes all Collections via child links (but has no items at all)?

Yes, I think so. I'm reluctant to add this as an example because it seems like a very uncommon case.